Makefile

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# No individual commands at the moment, just packages.
+CMDS=
+all: build test

```

Comment:
This will build as well as run the tests. I am not sure if this will be redundant. If it is redundant this line can be replaced with `all: build`

Reply:
I think the usual expectation is that a plain "make" will just build, i.e. use `all: build` here.

Reply:
Thank you for the info. I will update the PR :)